sdk/testcontainers-vault/src/vault.ts

```diff
@@ -21,20 +21,28 @@ export class Vault {
       bindMounts = []
     } = this.options;
 
+    let pvaultServerOrDev = ''
+    if (typeof (this.options.isPvaultServer) == 'undefined') {
+      pvaultServerOrDev = `piiano/pvault-dev:`
+    }
+    else {
+      pvaultServerOrDev = 'piiano/pvault-server:'
+    }
+
     if (!license) {
       throw new Error('Missing Vault license');
     }
 
     const vars = Object.entries(env).reduce((acc: Record<string, string>, [key, value]) => {
       acc[key] = String(value);
       return acc;
-    }, {PVAULT_SERVICE_LICENSE: license});
+    }, { PVAULT_SERVICE_LICENSE: license });
 
-    this.container = new GenericContainer(`piiano/pvault-dev:${version}`)
-      .withExposedPorts(port ? {container: vaultPort, host: port} : vaultPort)
+    this.container = new GenericContainer(`${pvaultServerOrDev}${version}`)
```

Comment:
Set `:` in the string and not as part of the image name.
```suggestion
    this.container = new GenericContainer(`${vaultImage}:${version}`)
```